SPEC.md

```diff
@@ -569,34 +569,70 @@ JSON Schema for the Container Runtime Manifest (container manifest), conforming
     "uuid": "6733C088-A507-4694-AABF-EDBE4FC5266F",
     "apps": [
         {
-            "app": "example.com/reduce-worker-1.0.0",
-            "imageID": "sha512-...",
+            "name": "reduce-worker",
+            "image": {
+                "name": "example.com/reduce-worker",
+                "id": "sha512-...",
+                "labels": [
+                    {
+                        "name":  "version",
+                        "value": "1.0.0"
+                    }
+                ]
+            },
+            "app": {
+                "exec": [
+                    "/bin/reduce-worker",
+                    "--debug=true",
+                    "--data-dir=/mnt/foo"
```

Comment:
weird how? like out of place because you expect them to be space separated?

Or weird as in rendered weirdly?

If it's the latter, this is just one of those examples of why anti-aliased text isn't really well-suited for small font sizes unless your DPI is very high.  If your environment is using anti-aliasing then two perfectly horizontal bars of 100% opacity pixels will stand out in a sea of fuzzy italicized text having anti-aliased edges, since the equals sign can often be perfectly rendered on a pixel array while the rest are misaligned fractional pixels.